include/robot_localization/ros_filter.hpp

```diff
@@ -245,6 +245,31 @@ class RosFilter : public rclcpp::Node
   //!
   void periodicUpdate();
 
+  //! @brief Update filter with data from measurements queue.
+  //! @param[in] time - The time at which to carry out integration.
+  //!
+  void updateFilterWithMeasurements(const rclcpp::Time & time);
+
+  //! @brief publish world to base link transform and position
+  //! with odometry message
+  //! @param[in] filtered_position - filtered position from
+  //! getFilteredOdometryMessage.
+  //! @return true if data is corrected otherwise false.
+  //!
+  bool publishPositionWithOdometry(nav_msgs::msg::Odometry filtered_position);
```

Comment:
If we're sticking with reference types, is there any reason this isn't const ref?